src/app/router.js

```diff
@@ -68,6 +69,7 @@ class AppRouter extends Router {
         this.route('give-confirmation', 'give');
         this.route('interest-confirmation', 'confirmation');
         this.route('higher-ed', 'home');
+        this.route('institutional-partnership-application', 'institutional-partnership');
```

Comment:
You shouldn't need this line. This section is for special routes. The entry on line 39 handles the regular routing.